draft-ietf-rtcweb-jsep.xml

```diff
@@ -2089,9 +2089,15 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               direction:
               <list style="symbols">
 
-                <t>An "a=msid" line, as specified in
-                <xref target="I-D.ietf-mmusic-msid"></xref>, Section
-                2.</t>
+                <t>For each MediaStream that was associated with the
+                transceiver when it was created via addTrack or addTransceiver,
+                an "a=msid" line, as specified in
+                <xref target="I-D.ietf-mmusic-msid"></xref>, Section 2. If a
+                MediaStreamTrack is attached to the transceiver's RtpSender,
+                the "a=msid" lines MUST use that track's ID. If no
+                MediaStreamTrack is attached, a valid ID MUST be generated,
+                in the same way that the implementation generates IDs for
```

Comment:
I don't see any problem with that. The track IDs really only have value to applications using "addTrack", since if you're using the transceiver API you can use MIDs.

Along those lines, I guess another option would be to say "Always generate the ID, unless a track was attached to the transceiver via addTrack".